inst/assets/bibliography.bib

```diff
@@ -2290,4 +2290,21 @@ @article{Hang2021
     year = {2021},
     issn = {1474-760X},
     doi = {10.1186/s13059-021-02368-1},
-}
\ No newline at end of file
+}
+
+@Manual{R_iSEEtree,
```

Comment:
we should stick to LastnameYear format (and if necessary LastnameYearInfo, like Benedetti2024iseetree